www/js/version_check.js

```diff
@@ -2,34 +2,31 @@
  * Checks if the API version in the app "appAPIVersion" matches the
  * web API version and if not, prompts a popup to force the user to update.
  */
-function checkAPIVersion(appAPIVersion) {
-  $.getJSON(API + '/versions')
-    .done(function(resp) {
-      if (resp.current_version !== appAPIVersion) {
+function checkAPIVersion() {
+  const appAPIVersion = 1.1;
```

Comment:
Thinks this name makes more sense.
```suggestion
  const localAPIVersion = 1.1;
```